testing/influxdb/APKBUILD

```diff
@@ -0,0 +1,72 @@
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=influxdb
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="Scalable datastore for metrics, events, and real-time analytics"
+url="https://www.influxdata.com/time-series-platform/influxdb/"
+arch="all"
+license="MIT"
+makedepends="go asciidoc xmlto"
+pkgusers="influxdb"
+pkggroups="influxdb"
+install="$pkgname.pre-install"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.zip::https://github.com/influxdata/$pkgname/archive/v$pkgver.zip
+	influxdb.initd
+	influxdb.logrotate"
+builddir="$srcdir/$pkgname-$pkgver"
+options="!strip"
+
+_collectdver=0.2.0
+
+build() {
+	export GOPATH="$srcdir/gopath"
+
+	mkdir -p "$GOPATH/src/github.com/influxdata"
+	ln -s "$builddir" "$GOPATH/src/github.com/influxdata/influxdb"
+
+	cd "$GOPATH/src/github.com/influxdata/influxdb"
+	go get -v -d ./... || return 1
+
+	git --git-dir="$GOPATH/src/collectd.org/.git" \
+		--work-tree="$GOPATH/src/collectd.org" \
+		checkout -f tags/v"$_collectdver" || return 1
```

Comment:
Because as you said, I don't use vendoring properly and the version on master isn't compatible with influxdb 1.1.0. Try to work out a glide based solution now...